### ISSUE TITLE
4.2.2 / 32.2.1 and PRs

### ISSUE THREAD

**Issue by lhammond** [idavis/jetson-containers#43]
First of all, @idavis this work you've done is straight fire.

The documentation is extremely well written.  I'm working through the process now and it's making a lot of sense.  I hope to test on a nano within the next few hours.

I'm targeting 4.2.2 / 32.2.1 Tensorflow and the Dockerfiles in the zoo example weren't there.
I just copied the 32.2 min-full to the correct naming convention.

CUDA_PKG_VERSION wasn't getting populated so I found the file in `/tmp/GA_4.2.2/P3448-0000/cuda-repo-l4t-10-0-local-10.0.326_1.0-1_arm64.deb` and created `ARG CUDA_PKG_VERSION=10.0.326_1.0-1` at the top of my new `32.2.1-nano-dev-jetpack-4.2.2-min-full.Dockerfile`.

It's building now.

I did see this log message scroll by : `Configuration file /etc/apt/sources.list.d/cuda-10-0-local-10.0.326.list, does not exist on system.  Installing new config file as you requested.`

couple of questions:
- do you see any issues with this approach?
- do you think I should be concerned about the above log message?
- is your goal to track version changes with the zoo examples?  If so, would you like me to PR the additional Dockerfiles?
- if your goal is to track version changes, perhaps I could start looking at testing 4.4.1 for a possible PR?  Any caveats to this?

again, kudos for the great work .. very thoughtful and well done.
cheers

**Comment by idavis** [comment]
@lhammond  Sorry I just saw this and other issues, somehow I didn't get a notification. I'll take a look this weekend if you're still having issues.

**Comment by idavis** [comment]
@lhammond I'm trying to build TF containers into the 4.4 PR #45 . This will generate base containers for all available TF versions.

**Comment by idavis** [comment]
@lhammond I have pushed ab1afdd9cd95f5a94dba22a86231ea3160b728fd in #45 which adds TF support. I have baked in TF base images from which a TF sample can be created. I was going to use the original TF sample and then add tensorrt support, but I'll track that through another issue. 

Tracking changes in zoo itself is just too much work. I have generated tasks and images for all valid TF builds published by NVIDIA.

Do not worry about the log message. It will be in every build.

I'm going to wrap up the 4.4 support shortly and we can chat about what ideas you have for TF 👍 - thank you for the urging. I'm going to close this issue for now and we can track future TF enhancements separately.
